### ISSUE TITLE
npm install doesnt create package.json file and makes a global installation

### ISSUE THREAD

**Issue by lukasz21** [npm/npm#21176]
#### I'm opening this issue because:
 
  - [x ] npm is producing an incorrect install.
  - [x ] npm is doing something I don't understand.
  - [x ] npm is producing incorrect or undesirable behavior. 

#### What's going wrong?
 
_How it should be:_
I use `npm install underscore.string` command to make a local  installation in a folder without a package.json file.  In theory this should give the following result:

-  makes a local installation of package and CREATES a package.json file. 

 
_What I get:_
What I get as a result is this:

- makes a global installation of package and  DOSEN'T CREATE a package.json file.
I have the same issue with others packages.  

#### How can the CLI team reproduce the problem?
I'm not sure, but: I have the newest version of node.js [8.11.3 LTS] and npm [6.1.0].  And I still get this issue.

### supporting information:

 - `npm -v` prints: 8.11.3
 - `node -v` prints: 6.1.0
 - `npm config get registry` prints: https://registry.npmjs.org/
 - Windows, OS X/macOS, or Linux?: Windows 7 HP

**Comment by DanielRuf** [comment]
What does `npm config get` output?

**Comment by lukasz21** [comment]
` npm config get` shows:

```
; cli configs
metrics-registry = "https://registry.npmjs.org/"
scope = ""
user-agent = "npm/6.1.0 node/v8.11.3 win32 x64"

; builtin config undefined
prefix = "C:\\Users\\Admin\\AppData\\Roaming\\npm"

; node bin location = C:\Program Files\nodejs\node.exe
; cwd = C:\Users\Admin
; HOME = C:\Users\Admin
; "npm config ls -l" to show all defaults.
```

**Comment by DanielRuf** [comment]
Ok. Did you already try a clean reinstall of node and npm?

**Comment by lukasz21** [comment]
No, I will give it a try. But I will write about results later, because I'm finishing computer work for today.

**Comment by lukasz21** [comment]
Hi, so I tried to reinstall node and npm. It didn't change anything.
The same problem occurs.
'npm config get' command shows the same as before.

Don't know if it's important, but:
The problem occurs only when I try to make a local installation to a a folder without a package.json file. 
If I create in this folder package.json file, before I began to install locally a package, then the local installation works. Then the package is only in the folder.
